docs/src/main/asciidoc/datasource-guide.adoc

```diff
@@ -223,6 +223,16 @@ AgroalDataSource dataSource1;
 AgroalDataSource dataSource2;
 --
 
+== Datasource Health Check
+
+If you are using the `quarkus-smallrye-health` extension, `quarkus-agroal` will automatically add a readiness health check
+to validate the datasource.
+
+So when you access the `/health/ready` endpoint of your application you will have information about the datasource validation status.
+If you have multiple datasources, all datasources will be checked and the status will be `DOWN` as soon as there is one datasource validation failed.
```

Comment:
Change "validation failed" to "validation failure"

Reply:
done